core/includes/update.inc

```diff
@@ -294,6 +294,88 @@ function update_module_add_to_system($modules = array()) {
   }
 }
 
+/** 
+ * Checks for disabled dependencies during a Drupal 7 upgrade.
+ * 
+ * @return string
+ *    A themed message that lists modules that will be enabled,
+ *    to be displayed in the 'info' page of the update process. 
+ */
+
+function update_upgrade_check_dependencies() {
+  $status_report = '';
+  if (backdrop_get_installed_schema_version('system') > 7000) {
+    $modules_to_enable = update_upgrade_modules_to_enable();
+    
+    if (empty($modules_to_enable)) {
+      return '';
+    }
+
+    $files = system_rebuild_module_data();
+    $module_list = '';
+    foreach ($modules_to_enable as $module_to_enable => $dependents) {
+      $module_list .= '<strong>' . $files[$module_to_enable]->info['name'] . '. ' . t('Required by:') . '</strong>';
+      $module_list .= theme('item_list', array('items' => $dependents));
+    }
+    $status_report = '<h4>' . t('Some modules need to be enabled') . '</h4>';
+    $status_report .= t('The following Backdrop modules will be enabled in order for the site to be upgraded successfully:') . '</p>';
```

Comment:
I don't believe using `t()` is completely safe here. At the point we're upgrading from Drupal 7 to Backdrop, I don't think the translation system will be working yet. In English `t()` has no effect, but in other languages this will probably cause an error. This is called from `update_info_page()` which, just has all strings in English. I think all these `t()` calls should simply be removed.

Reply:
Got rid of t() in later commit. Thanks!